frontend/main.js

```diff
@@ -45,7 +45,7 @@ function runProgram(filePath) {
     OPTIONS.FORMAT,
     identifyFormat(fileName),
     filePath,
-    currentDirectory + "/",
+    currentDirectory + "/" + RESULTS_SPREADSHEET,
```

Comment:
rename this now that we support other formats?

Reply:
Changed this variable to `RESULTS_FILE`